client/src/views/TableDetail.vue

```diff
@@ -1,9 +1,17 @@
 <template>
   <div>
     <h1>Table: {{`${this.workspace}/${this.table}`}}</h1>
+
+<!-- 
     <ul>
-      <li v-for="field in fields" :key="field">{{field}}</li>
+     <li v-for="field in fields" :key="field">{{field}}</li> 
+       <li v-for="row in rowKeys" :key="row[0].value">{{row[0].value}}</li>
     </ul>
+-->
```

Comment:
If we're really dumping this code that's been commented out, I'd prefer if we actually cut it out altogether.